### PR TITLE
test: move creation and deletion and resources outside of `It`

### PR DESCRIPTION
**Pull request by ianvernon**
Fix tests in the following files which have this issue:

* `test/runtime/cli.go`
* `test/runtime/Policies.go`
* `test/runtime/monitor.go`

Signed-off by: Ian Vernon <ian@cilium.io>

Relates-to: #3821

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: test/runtime/cli.go [hunk 1]
```diff
@@ -55,15 +48,26 @@ var _ = Describe("RuntimeValidatedCLI", func() {
...
-		It("Test labelsSHA256", func() {
-			fooID := "id.foo"
-			namesLabels := [][]string{{"foo", fooID}, {"bar", "id.bar"}, {"baz", "id.baz"}}
 
+		var (
+			fooID       = "id.foo"
+			namesLabels = [][]string{{"foo", fooID}, {"bar", "id.bar"}, {"baz", "id.baz"}}
+		)
+
+		BeforeEach(func() {
```
Comment (eloycoto) [inline test/runtime/cli.go]: This can be BeforeAll/AfterAll no?

**Review by eloycoto — DISMISSED** [review]

File: test/runtime/cli.go (hunk 1 above)
Comment (ianvernon) [reply test/runtime/cli.go]: True, I'll fix that.

Comment (ianvernon) [reply test/runtime/cli.go]: Fixed in latest upload.

**Comment by ianvernon** [comment]
test-me-please

**Comment by eloycoto** [comment]
Awesome!

**Review by eloycoto — APPROVED** [review]

**Pull request merged**